frontend/views/utils/ui.js

```diff
@@ -2,15 +2,35 @@
 
 import sbp from '@sbp/sbp'
 import { OPEN_MODAL, MODAL_RESPONSE } from '@utils/events.js'
+import L, { LError } from '@view-utils/translations.js'
+
+// Call from anywhere in the app:
```

Comment:
The comment should mention that to use this `BANNER` needs to be set, and maybe give an example

Reply:
A few usage exemples can be found just below. Do you mean there should be an example of explicitly setting BANNER before use?
By the way, the related selectors can still be used somewhat safely even if `BANNER` is not set yet, thanks to the `?.` operator.

Reply:
>  Do you mean there should be an example of explicitly setting BANNER before use?

Yeah, I think that'd be useful to mention

> By the way, the related selectors can still be used somewhat safely even if BANNER is not set yet, thanks to the ?. operator.

Yeah I saw, nice one! 👍